samples/bluetooth/peripheral_nucleo_wb55rg/README.rst

```diff
@@ -0,0 +1,30 @@
+.. _bluetooth-peripheral_nucleo_wb55rg:
+
+Bluetooth: STM32WB55 BLE peripheral demo
+########################################
+
+Overview
+********
+
+This application demonstrates BLE peripherals on an `ST Nucleo WB55RG`_
+by exposing a battery service and vendor-specific GATT services.
+
```

Comment:
Application proposes a variety of use cases, mostly towards sensors.
So please expose here what are the services currently implemented.

Reply:
done

Reply:
This application demonstrates BLE peripherals on an `ST Nucleo WB55RG`_
by exposing vendor-specific GATT services.
Currently only button notification and LED service are implemented.